### PR TITLE
Use CDAP 4.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by wolf31o2**
The `develop` branch here should match `develop` of CDAP.

### CODE REVIEW

**Comment by wolf31o2** [comment]
Build: http://builds.cask.co/browse/NAV-DA7-3

File: pom.xml [hunk 1]
```diff
@@ -92,6 +92,10 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```
Comment (anwar6953) [inline pom.xml:95]: why is this needed only now?

Comment (wolf31o2) [reply pom.xml:95]: AFAIK, it looks like it's always been needed and it's been only because the local build agents already had snapshots in their Maven cache that it worked.

**Comment by anwar6953** [comment]
LGTM
